src/site/content/en/progressive-web-apps/what-are-pwas/index.md

```diff
@@ -0,0 +1,135 @@
+---
```

Comment:
Changing the document title from `what-are-pwas` to `what-are-progressive-web-apps` might give you better SEO (depending on what phrase you want to target), I'm pretty sure that document titles do have a noticeable effect on documentation SEO in general.